fixtures/default.json

```diff
@@ -467,23 +467,23 @@
         },
         "variable": {
           "type": "str",
-          "description": "The variable on which this loop operates. This parameter is used to rename the input and outputs of this module for easy integration for the inputs and outputs of firmware modules. For example if the `variable` were `air_temperature`, `desired` would become `air_temperature_desired`, `state` would become `air_temperature`, `cmd` would become `air_temperature_cmd`, `up_cmd` would become `air_temperature_up_cmd`, and `down_cmd` would become `air_temperature_down_cmd"
+          "description": "The variable on which this loop operates. This parameter is used to infer which ROS Topics are used for measurements from and commands to firmware modules."
         }
       },
       "inputs": {
         "desired": {
           "type": "std_msgs/Float64",
-          "description": "The message data element must contain the desired value of the state measurement of the controlled process. This topic can be renamed by the variable parameter"
+          "description": "The message data element must contain the desired value of the state measurement of the controlled process."
         },
         "state": {
           "type": "std_msgs/Float64",
-          "description": "The message data element must contain the current value of the controlled plant property. The controller publishes std_msgs/Float64 messages on the control_effort topic each time it receives a message on the state topic. Thus the rate at which the plant publishes state governs the control-loop rate - the plant should publish state at the desired loop rate. This topic can be renamed by the topic_from_plant parameter"
+          "description": "The message data element must contain the current value of the controlled plant property. The controller publishes messages on the commanded topic each time it receives a message on the state topic. Thus the rate at which the plant publishes state governs the control-loop rate - the plant should publish state at the desired loop rate."
         }
       },
       "outputs": {
         "cmd": {
-          "type": "std_msgs/Float64",
-          "description": "The control_effort message data element contains the control effort to be applied to the process to drive state/data to equal setpoint/data. This topic can be renamed by the topic_from_controller parameter"
```

Comment:
Couldn't find any references to `topic_from_controller` in the code, but let me know if this shouldn't be removed